src/runtime/GlobalObjectBuiltinArray.cpp

```diff
@@ -1489,6 +1489,22 @@ static Value builtinArrayShift(ExecutionState& state, Value thisValue, size_t ar
     return first;
 }
 
+static Value builtinArrayUnscopables(ExecutionState& state, Value thisValue, size_t argc, Value* argv, bool isNewExpression)
+{
+  Value blacklist = state.context()->globalObject()->objectCreate()->call(state, Value(), 1, &blacklist);
```

Comment:
Isn't it too complicated way to create an empty object?

Reply:
Otherwise, the approach is incorrect, since according to the standard the blackList object's prototype should be null.

```
Object * blackList = new Object(state);
blackList->setPrototype(state, nullptr);
```

After that all `blackList.asObject()` casts become unnecessary.